hedera-mirror-test/src/test/java/com/hedera/mirror/test/e2e/acceptance/steps/AccountFeature.java

```diff
@@ -77,6 +81,31 @@ public void treasuryDisbursement(long amount, String accountName) {
         assertNotNull(networkTransactionResponse.getReceipt());
     }
 
+    @Given("I send {long} tℏ from {string} to {string} alias not present in the network")
+    public void createAccountOnTransferForEDAlias(long amount, String senderAccount, String keyType) {
+        senderAccountId = accountClient.getAccount(AccountClient.AccountNameEnum.valueOf(senderAccount));
+        startingBalance = accountClient.getBalance(senderAccountId);
+
+        var recipientPrivateKey =  "ed".equalsIgnoreCase(keyType) ? PrivateKey.generateED25519() : PrivateKey.generateECDSA();
+        receiverAccountId = recipientPrivateKey.toAccountId(0,0);
+
+        networkTransactionResponse = accountClient.sendApprovedCryptoTransfer(senderAccountId, receiverAccountId, Hbar.fromTinybars(amount));
```

Comment:
once we change the test case to send from the client operator

```suggestion
        networkTransactionResponse = accountClient.sendCryptoTransfer(receiverAccountId, Hbar.fromTinybars(amount));
```

Reply:
fixed